api/base/urls.py

```diff
@@ -19,8 +20,16 @@
                 url(r'^applications/', include('api.applications.urls', namespace='applications')),
                 url(r'^collections/', include('api.collections.urls', namespace='collections')),
                 url(r'^comments/', include('api.comments.urls', namespace='comments')),
-                url(r'^docs/', include('rest_framework_swagger.urls')),
+                url(r'^nodes/', include('api.nodes.urls', namespace='nodes')),
```

Comment:
See L12 and please keep this list alphabetized (unless the new version of DRS auto-alphabetizes regardless if this list order, in which remove L12).